pavelib/assets.py

```diff
@@ -384,14 +384,24 @@ def coffeescript_files():
 
 @task
 @no_help
-def compile_coffeescript(*files):
+def compile_coffeescript(*coffee_files):
     """
     Compile CoffeeScript to JavaScript.
     """
-    if not files:
-        files = ["`{}`".format(coffeescript_files())]
+    js_files = ""
+
+    if not coffee_files:
+        coffee_files = ["`{}`".format(coffeescript_files())]
+        js_files = r"`{} | sed 's/\.coffee/\.js/g'`".format(coffeescript_files())
+    else:
+        js_files = r"`{} | sed 's/\.coffee/\.js/g'`".format(*coffee_files)
```

Comment:
We need a list of every coffeescript file (to convert them to javascript) and then we need a list of all the javascript files that were converted in order to add this header to them. Can't get that without shelling out, unfortunately, because you have to actually run the `coffeescript_files` command to find the CS files before changing their extensions to JS - just running that same search for JS files will turn up lots of things that weren't converted from coffeescript.